server/src/main/resources/org/opensearch/bootstrap/test-framework.policy

```diff
@@ -128,12 +128,16 @@ grant codeBase "${codebase.httpasyncclient}" {
   permission java.net.NetPermission "getProxySelector";
 };
 
-
 grant codeBase "${codebase.junit-rt.jar}" {
   // allows IntelliJ IDEA JUnit test runner to control number of test iterations
   permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
 };
 
+grant codeBase "${codebase.idea_rt.jar}" {
+  // allows (newer versions of) IntelliJ IDEA JUnit test runner to control number of test iterations
```

Comment:
nit: Over time `(newer versions of)` will not be applicable. May be we can remove it ?

Reply:
It'll always be newer relative the version of IDEA referred to by line 132 :)

I'll make this more precise.